src/Checkpointer.jl

```diff
@@ -126,11 +126,13 @@ function restore_cache!(sim::Interfacer.ComponentModelSimulation, new_cache)
 end
 
 """
-    checkpoint_sims(cs::CoupledSimulation)
+    checkpoint_sims(cs::CoupledSimulation, save_cache)
 
-This is a callback function that checkpoints all simulations defined in the current coupled simulation.
+This is a callback function that checkpoints all simulations defined in the
+current coupled simulation. The argument `save_cache` determines whether to save
+the cache or not.
 """
-function checkpoint_sims(cs::Interfacer.CoupledSimulation)
+function checkpoint_sims(cs::Interfacer.CoupledSimulation, save_cache)
```

Comment:
It would be nice to maintain the interface of `checkpoint_sims` taking in only `cs`. Could we store `save_cache` in there instead?